app/views/product_users/_table.html.haml

```diff
@@ -20,6 +21,7 @@
         %td
           = link_to user.last_first_name, [current_facility, user]
 
+        %td= product_user.approved_at.strftime("%Y/%M/%d")
```

Comment:
`%M` is going to display minutes, `%m` is for month.  Also, most other dates in the app are displaying the year at the end, so let's go for consistency:
```suggestion
        %td= product_user.approved_at.strftime("%m/%d/%Y")
```